package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "repository": {
-  },
+  "repository": {},
   "dependencies": {
-    "brunch": "^1.8.5",
     "babel-brunch": "^5.1.1",
+    "brunch": "^1.8.5",
     "clean-css-brunch": ">= 1.0 < 1.8",
     "css-brunch": ">= 1.0 < 1.8",
     "javascript-brunch": ">= 1.0 < 1.8",
+    "sass-brunch": "^1.9.1",
```

Comment:
Please check this file as an example: https://github.com/pragmaticivan/hex_web/blob/feature/new-frontend/web/static/css/template/_footer.scss